### ISSUE TITLE
wordpress.com blogs credentials

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#638]
```
What's the problem?
I'm experiencing problems to log in to my WP.com accounts by safe access.
Meanwhile log to my WP account is correctly working.





What browser are you using?
Opera Next / Chrome 20.0.1132.47 m





What version of ScribeFire are you running?
Opera extension / Chrome extension

```

-----
Original issue reported on code.google.com by `marco.b...@gmail.com` on 2 Jul 2012 at 1:36

Attachments:
* [2012-07-02_153401.png](https://storage.googleapis.com/google-code-attachments/scribefire-chrome/issue-638/comment-0/2012-07-02_153401.png)

**Comment by GoogleCodeExporter** [comment]
```
I have the same issue, but in Firefox 17.0.1. Going back to ScribeFire classic 
solves the problem.
```

-----
Original comment by `jnagyjr1...@gmail.com` on 18 Dec 2012 at 5:09

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in ScribeFire version 4.2. This should be available in the 
Chrome, but still waiting on approval from Firefox & Opera.
```

-----
Original comment by `scribefi...@gmail.com` on 3 Sep 2013 at 9:48
* Changed state: **Fixed**
